File: uri/bonus/bonus.java
import java.io.IOException;
import java.util.*;

public class Main
{
    public static double meubonus( double salario, double montante )
    {
        return ( salario + ((montante * 15)/100));
    }

    public static void main( String[] args ) throws IOException
    {
        Scanner input = new Scanner(System.in);
        String nome;
        Double salario, montante;
        nome = input.nextLine();
        salario = input.nextDouble();
        montante = input.nextDouble();
        System.out.printf("TOTAL = R$ %.2f\n", meubonus(salario, montante));
        input.close();
    }
}

File: uri/bonus/bonus.kt
import java.util.*
import kotlin.text.*

fun pagamento ( salario: Double, montante : Double) : Double{
    return ( salario + ((montante * 15)/100));
}

fun main(args:Array<String>){
    var input = Scanner(System.`in`)
    var salario : Double; var montante : Double; var resultado : Double; var nome : String
    nome = input.nextLine()
    salario = input.nextDouble()
    montante = input.nextDouble()
    resultado =  pagamento(salario, montante)

    println("TOTAL = R$ %.2f".format(resultado)) 
}